pylint_odoo/test_repo/broken_module/models/broken_model.py

```diff
@@ -227,6 +227,21 @@ def my_method1(self, variable1):
         self.message_post(body=variable1 % variable1)
         self.message_post(body=(variable1 % variable1))
 
+        # translation function with variables in the term
+        variable2 = variable1
+        self.message_post(_('Variable not translatable: %s' % variable1))
+        self.message_post(_('Variables not translatable: %s, %s' % (
+            variable1, variable2)))
+        self.message_post(body=_('Variable not translatable: %s' % variable1))
+        self.message_post(body=_('Variables not translatable: %s' % (
+            variable1, variable2)))
+        error_msg = _('Variable not translatable: %s' % variable1)
+        error_msg = _('Variables not translatable: %s, %s' % (
+            variable1, variable2))
+        error_msg = _('Variable not translatable: {}'.format(variable1))
+        error_msg = _('Variables not translatable: {}, {variable2}'.format(
+            variable1, variable2=variable2))
```

Comment:
@pedrobaeza 
Do you think that's necessary?

There are plenty of well-former cases, see [here](https://github.com/OCA/pylint-odoo/blob/8845b74/pylint_odoo/test_repo/broken_module/models/broken_model.py#L206-L219). That's the reason why I didn't included them, because they are already covered in the same method.

Reply:
OK, in that case indeed it's not needed. Check Travis any way, as it's now red.